### ISSUE TITLE
Add option to display alive + open ports

### ISSUE THREAD

**Issue by labsy** [angryip/ipscan#102]
Hi,
It would be great if you could add an option to display not only alive OR open ports, but also "Display Alive and hosts with open ports".
Why? Because some machines respond to PING, but have no open ports (at least not those, which I scan against), and also there are some machines, which do not respond to PING, but have some ports open. So the FULL results are those with open ports + those who respond to pings.
Hope this makes sense.
Thanx!

**Comment by angryziber** [comment]
If you mean that you still want to still continue scanning hosts, which don't respond to pinging, then enable "Scan dead hosts, which don't reply to pings" in Preferences dialog, Scanning tab.

**Comment by labsy** [comment]
Hi,
no, I do not mean that. I mean that results are displayed:
- either ALL IP addresses, no filter
- or only those, responding to PINGs
- or only those with open ports
What I mean is to add an option to display results:
- responding to PINGs or have open ports

As of now there is no way to display BOTH. Now I need to run scanner 2 times, once to display open ports and once again to display those, which respond to pings, so only then I can be sure some particular IP is not already taken.
Hope this time I made it more clear :)

**Comment by angryziber** [comment]
Ok, displaying only "Alive hosts" usually implies both.
But I guess you have "Scan dead hosts, which don't reply to pings" enabled?

**Comment by labsy** [comment]
Yes, I have "Scan dead hosts, which don't reply to pings" enabled.
But if some PC has firewall set to block ICMP ping and at the same time it has some ports opened, this machine is NOT displayed if I have selected to display only "Alive hosts".
